wazo_webhookd/services/mobile/plugin.py

```diff
@@ -317,7 +320,7 @@ def _create_apn_message(self, message_title, message_body, channel_id, data):
             }
         elif channel_id == 'wazo-notification-cancel-call':
             headers = {
-                'apns-topic': 'io.wazo.songbird',
+                'apns-topic': apns_topic_value,
```

Comment:
This one should use the `mobile_apns_default_topic` since it used `mobile_apns_default_topic` before